src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -42,6 +42,14 @@ internal abstract class PENamespaceSymbol
         /// </summary>
         private Dictionary<string, TypeDefinitionHandle> _lazyNoPiaLocalTypes;
 
+        /// <summary>
+        /// Caches the last call to GetMembers(string) when that member is a namespace. Multiple types
+        /// within the same namespace are often requested in sequence, so this allows the second and
+        /// subsequent request for members with the same name to share a result without looking up the
+        /// values or allocating.
+        /// </summary>
+        private ImmutableArray<Symbol> _lastGetMembersResult;
```

Comment:
>_lastGetMembersResult [](http://example.com/codeflow?start=39&length=21)

Consider using member name that more closely reflects what it is used for. For example, it looks like the array might have absolutely no relation to the last GetMemebers result.

Reply:
@AlekseyTs Done. Let me know if you have a better name.